File: modul 3/Bahasa C/PRAK305-2210817110014-Muhammad_Firas.c
#include<stdio.h>

int main() 
{
    int angka;
    scanf("%d", &angka);
    int hari = angka / 86400;
    int sisa_hari = angka % 86400;
    int jam = sisa_hari / 3600;
    int sisa_jam = angka % 3600; 
    int menit = sisa_jam / 60;
    int detik = angka % 60;

    if (angka >= 86400) {
        printf("%d hari %d:%d:%d", hari,jam,menit,detik);
    }
    else if (angka > 20000) {
        printf("0%d:%d:0%d",jam,menit,detik);
    }
    else if (angka > 3600) {
        printf("0%d:%d:%d",jam,menit,detik);
    }
    else if (angka == 3600) {
        printf("0%d:0%d:0%d",jam,menit,detik);
    }
    else {
        printf("00:%d:%d", menit, detik);
    }
}

File: modul 3/Bahasa Python/PRAK305-2210817110014-Muhammad Firas.py
angka = int(input(""))

hari = angka // 86400
sisa_hari = angka % 86400
jam = sisa_hari // 3600
sisa_jam = angka % 3600 
menit = sisa_jam // 60
detik = angka % 60

if (angka >= 86400):
    print("{} hari {}:{}:{}".format(hari, jam, menit, detik))

elif (angka >= 9000 <86400):
    print("0{}:{}:0{}".format(jam,menit,detik))

elif (angka > 3600 <9000):
    print("0{}:{}:{}".format(jam,menit,detik))

elif (angka == 3600):
    print("0{}:00:00".format(jam))
 
else: 
    print("00:{}:{}".format(menit,detik))